build/tests/CMakeLists.txt

```diff
@@ -48,3 +48,14 @@ add_executable("ut"
   ${TESTS_SOURCES}
 )
 target_link_libraries("ut" "gtest" ${TESTS_LIBRARIES} ${NEB} "rokerbase" "roker")
+
+add_test(tests ut)
```

Comment:
Please use add_test(NAME tests COMMAND ut) for the automatic binary path replacement (https://cmake.org/cmake/help/v3.9/command/add_test.html).